catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -0,0 +1,279 @@
+    }
+
+    public static List<ServiceType> getServices(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, ServiceType.class);
+    }
+
+    public static List<ServiceType> getServices(RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, ServiceType.class);
+    }
+
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, ExtrinsicObjectType.class);
+    }
+
+    public static List<ExtrinsicObjectType> getExtrinsicObjects(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, ExtrinsicObjectType.class);
+    }
+
+    public static List<OrganizationType> getOrganizations(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, OrganizationType.class);
+    }
+
+    public static List<OrganizationType> getOrganizations(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, OrganizationType.class);
+    }
+
+    public static List<PersonType> getPersons(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, PersonType.class);
+    }
+
+    public static List<PersonType> getPersons(RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, PersonType.class);
+    }
+
+    public static List<AssociationType1> getAssociations(RegistryPackageType registryPackage) {
+        return getObjectsFromRegistryObjectList(registryPackage, AssociationType1.class);
+    }
+
+    public static List<AssociationType1> getAssociations(
+            RegistryObjectListType registryObjectList) {
+        return getObjectsFromRegistryObjectList(registryObjectList, AssociationType1.class);
+    }
+
+    public static <T extends RegistryObjectType> List<T> getObjectsFromRegistryObjectList(
+            RegistryPackageType registryPackage, Class<T> type) {
+        List<T> registryObjects = new ArrayList<>();
+
+        if (registryPackage == null) {
+            return registryObjects;
+        }
+
+        if (registryPackage.isSetRegistryObjectList()) {
+            registryObjects =
+                    getObjectsFromRegistryObjectList(registryPackage.getRegistryObjectList(), type);
+        }
+
+        return registryObjects;
+    }
+
+    public static <T extends RegistryObjectType> List<T> getObjectsFromRegistryObjectList(
+            RegistryObjectListType registryObjectList, Class<T> type) {
+        List<T> registryObjects = new ArrayList<>();
+
+        if (registryObjectList == null) {
+            return registryObjects;
+        }
+
+        registryObjects.addAll(registryObjectList.getIdentifiable()
```

Comment:
We shouldn't need the `registryObjects` `ArrayList` in this method and be able to return the result of the `.collect()` directly.